src/network/networkd-network.c

```diff
@@ -306,7 +306,7 @@ int network_verify(Network *network) {
 
         LIST_FOREACH_SAFE(prefixes, prefix, prefix_next, network->static_route_prefixes)
                 if (section_is_invalid(prefix->section))
-                        prefix_free(prefix);
+                        route_prefix_free(prefix);
```

Comment:
Longer term, we should make this type safe. Having two functions which look very similar and take the same type but are not interchangeable is asking for trouble.

Reply:
Yeah, I will do that in another PR. Thanks.

Reply:
#13944.